examples/resources/awscc_amplify_branch/multi_branch.tf

```diff
@@ -0,0 +1,41 @@
+resource "awscc_amplify_app" "example" {
+  name = "app"
+}
+
+resource "awscc_amplify_branch" "main" {
+  app_id      = awscc_amplify_app.example.app_id
+  branch_name = "main"
+
+  framework = "React"
+  stage     = "PRODUCTION"
+
+  environment_variables = [
+    {
+      name  = "REACT_APP_API_SERVER"
+      value = "https://api.example.com"
+    },
+    {
+      name  = "Environment"
+      value = "PROD"
+    },
+  ]
+}
+
+resource "awscc_amplify_branch" "dev" {
+  app_id      = awscc_amplify_app.example.app_id
+  branch_name = "main"
```

Comment:
Would it be better to use development branch name as an example?

Reply:
yea, copy/paste error sorry - will fix now

Reply:
Fixed branch name typo and re-ran `make docs` command